### DISCUSSION TITLE
Welcome to SHiELD_build Discussions!

### DISCUSSION THREAD

**Discussion by laurenchilutti** [NOAA-GFDL/SHiELD_build#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lharris4** [comment]
Hi, everyone! I am Lucas Harris, head of GFDL's FV3 team. We are excited to make SHiELD available to the broader community and looking forward to seeing what our external partners or other interested people can do with this model.
